src/components/Identity/identityUtil.js

```diff
@@ -0,0 +1,61 @@
+import { AUTH_STATES } from "./constants";
+import isObject from "../../utils/isObject";
+
+const validateCustomerIDs = customerIDs => {
+  if (!isObject(customerIDs)) {
+    throw new Error(
+      "Invalid customer ID format. Each namespace should be an object"
```

Comment:
Let's extract those messages into a single variable to save those precious bits.